client/state/plans/interval-types.jsx

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
```

Comment:
Let's remove this comment if we have no external deps.

Reply:
es-lint complains when there's no comment :-(

Reply:
We should totally fix that; it shouldn't enforce it when there are no external imports.

Reply:
🤔 Are you sure it complains? This seems to have no errors:

```
/** @format */

/**
 * Internal dependencies
 */
import Button from 'components/button';

Button();
```

Reply:
Just removed this comment - looks like I got bamboozled by a different es-lint message :-)